src/HealthChecks.RabbitMQ/RabbitMQHealthCheck.cs

```diff
@@ -11,12 +11,18 @@ public class RabbitMQHealthCheck
     {
         private readonly IConnectionFactory _connectionFactory;
         private IConnection _rmqConnection;
+        protected bool AttemptConnectionReuse = false;
```

Comment:
```suggestion
        protected bool _attemptConnectionReuse;
```

Reply:
There is one other `protected` field in this repository and it uses PascalCasing:
https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/bc0f481404412015aa11aeb30709eebba5dcd9dc/src/HealthChecks.Publisher.Prometheus/LivenessPrometheusMetrics.cs#L14

Since there is no `.editorconfig` to specify the desired convention, I've just matched the existing precedent.

Reply:
OK, then `protected bool AttemptConnectionReuse;`